style.css

```diff
@@ -50,23 +67,25 @@
   word-spacing: 3px;
   color: #7f8cff;
   margin-left: 24px;
-  margin-top: 28px;
+  margin-top: 10px;
   margin-bottom: 10px;
 }
 
 .networks {
   text-decoration: none;
   margin: none;
+  margin-bottom: 120px;
   color: #505f79;
   list-style-type: none;
   display: flex;
   flex-direction: row;
   align-items: flex-start;
   padding: 24px;
-  padding-top: 10px;
   gap: 12px;
 }
 
-.linkedin {
-  color: solid #505f79;
+.fa-linkedin-in {
+  color: #505f79;
+  width: 18px;
+  height: 20px;
 }
```

Comment:
- [ ] Linkedin icon is still not fixed. Kindly use this [link](https://drive.google.com/file/d/1nrZf71ka2ymqqnItTz-wG4j7ziEUkSm6/view?usp=share_link) to download the png file and fix the issue as per previous reviewers suggestion.

  ![image](https://user-images.githubusercontent.com/60083437/219510858-3e0bafde-90fb-47a6-9911-90cd38072ff5.png)